### ISSUE TITLE
Bugs when changing python interpreter

### ISSUE THREAD

**Issue by rixez** [spyder-ide/spyder#13026]
### What steps will reproduce the problem?

Spyder 4.1.3 throws an internal problem error when I was trying to change the python interpreter of the spyder installed in environment A to the python interpreter in environment B. The error happenned when I restarted the kernel. Even though there is an error report, the console works as intended so its not a big problem.

### Traceback
```python-traceback
Exception in comms call _send_comm_config:
  File "J:\pydicomenv_ssd\lib\site-packages\spyder_kernels\comms\commbase.py", line 342, in _handle_remote_call
    buffer['call_kwargs'])
  File "J:\pydicomenv_ssd\lib\site-packages\spyder_kernels\comms\commbase.py", line 355, in _remote_callback
    raise CommError("No such spyder call type: %s" % call_name)
spyder_kernels.comms.commbase.CommError: No such spyder call type: _send_comm_config
```

## Versions

* Spyder version: 4.1.3 
* Python version: 3.7.6
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.4.1 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.5.2 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.10 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.4 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.0.4 (OK)
spyder_kernels >=1.9.1;<1.10.0 :  1.9.1 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.17 (OK)
matplotlib >=2.0.0             :  3.1.3 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  1.0.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5.1 (OK)
```

**Comment by goanpeca** [comment]
Hi @impact27, is this some sort of regression? Thanks

**Comment by impact27** [comment]
Maybe the version of spyder kernels doesn't correspond to the version of spyder?

**Comment by ccordoba12** [comment]
I think @impact27 is right.

@rixez, what's your spyder-kernels version in interpreter B?

**Comment by rixez** [comment]
@ccordoba12 After I restarted spyder and try the same thing again, the console's kernel would not start and prompt me to update the spyder-kernels. I couldn't remember exactly the previous version of spyder-kernels in interpreter B (maybe it was 1.3.x, I installed it with spyder 4.0). After the update, the version is 1.9.1 and there is no more error with spyder. So I think the problem is with the mismatch between spyder in environment A and spyder-kernels in environment B, as @impact27 pointed out.

Thank you for your help @goanpeca, @impact27, @ccordoba12!
